magpie/api/webhooks.py

```diff
@@ -188,8 +257,8 @@ def send_webhook_request(webhook_config, params, update_user_status_on_error=Fal
     except Exception as exception:
         LOGGER.error("An exception has occurred with the webhook request : %s", webhook_config["name"])
         LOGGER.error(str(exception))
-        if "user_name" in params and update_user_status_on_error:
-            webhook_update_error_status(params["user_name"])
+        if "user.name" in params and update_user_status_on_error:
```

Comment:
Hum, since we have group permission hook now, does this mean that we should have error status for groups too?
On the other side, having a permission webhook error does not mean that the user is "on error". The status was really intended for the creation webhook.
But then how to signal that permission webhooks are malfunctionning?

Reply:
> We could have that for group also, but for the moment, even the user permission change does not set a user-status error. Only the user create/update webhook change the status to error.
I think we would need a status on specific permissions, because as you said, setting an error status for the user or the whole group feels likes the "wrong target".

Yep I agree with the current state of things. 

> Another issue is how to report all those individual permissions status error in a useful way? Aggregate too much, and we don't know what is erroneous. Too granular, and we can never go through everything.

If I check the Magpie logs, will I get the required granularity to pinpoint the issue? If no, maybe you could log a little bit more on those issues. 
Then for the big picture, could we aggregate issues by webhook config? So that at least an admin, when logged, could see that some external service is misbehaving and be hinted that he should check the logs? (Not for this PR, but you could open a nice to have issue for Magpie)

Reply:
I think most of the errors would be hard to retrieve from logs, since part of the operation happens outside of magpie (the middleware subscribed in the webhook). For now, there is probably not even enough log details, but if it did, it could probably only say that the webhook replied with an error. 

What we could do is register the webhook failures in the db with relevant actions and configs. 
Those could be returned in a list, maybe filtered "by issue" similarly to how sentry does. I'm still not sure how granular per error cause would produce the best result.

Reply:
> only say that the webhook replied with an error

For me it's enough on the Magpie side. If Magpie can tell us that "service X" caused an error while creating a user, then we will know where to search.

I don't think we need a database for this right now. We don't need another sentry. Rather than going that route I would prefer if we could silently raise an error to sentry. But again this is out of scope for this PR.

Reply:
In that case, it is logged. It provides the `name` of the failing webhook and the "reason" of the http-error.